packages/plexus/src/zoom/MiniMap.tsx

```diff
@@ -81,10 +81,10 @@ function getClassNames(props: TProps) {
   };
 }
 
-export function MiniMap(props: TProps) {
-  const css = getClassNames(props);
-  const mapSize = getMapSize(props);
-  const activeXform = getViewTransform(props, mapSize);
+export function MiniMap({ className = '', classNamePrefix = 'plexus', ...props }: TProps) {
+  const css = getClassNames({ className, classNamePrefix, ...props });
+  const mapSize = getMapSize({ className, classNamePrefix, ...props });
+  const activeXform = getViewTransform({ className, classNamePrefix, ...props }, mapSize);
```

Comment:
these two functions do not rely on `className, classNamePrefix`, you can simply pass `props`